### PR TITLE
finger: add page

### PR DESCRIPTION
**Pull request by MasterOdin**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/finger.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# finger
+
+> User information lookup program.
+
+- Display the user's login name, real name, terminal name and write status (as a "\*" after the terminal name if write permission is denied), idle time, login time, office location and office phone number:
```
Comment (agnivade) [inline pages/common/finger.md]: We should start with the simplest invocation - `finger`. And probably shorten the description to `Display the user's info`

File: pages/common/finger.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+`finger -s`
+
+- Produce a multi-line format displaying all the information described for the option ```-s``` as well as the user's home directory, home phone number, login shell, mail status, and the contents of the files ".plan", ".project", ".pgpkey" and ".forward" from the user's home directory:
```
Comment (agnivade) [inline pages/common/finger.md]: This is copied verbatim from the man page. Please shorten it to a simpler version, and if possible use a word starting with `l`.

File: pages/common/finger.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+`finger -l`
+
+- Prevent the ```-l``` option of finger from displaying the contents of the ".plan", ".project" and ".pgpkey" files:
```
Comment (agnivade) [inline pages/common/finger.md]: I don't think this option is required to be in a tldr page.

Comment (MasterOdin) [reply pages/common/finger.md]: Removed.

File: pages/common/finger.md (hunk 2 above)
Comment (MasterOdin) [reply pages/common/finger.md]: Shortened it up, but no word came to mind that starts with an `l`.

File: pages/common/finger.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Display information about a specific user:
+
+`finger {{username}}`
+
+- Display the user's login name, real name, terminal name and write status (as a "\*" after the terminal name if write permission is denied), idle time, login time, office location and office phone number:
+
+`finger -s`
+
+- Produce multi-line format displaying same information as `-s` as well as user's home directory, home phone number, login shell, mail status as well as contents of files ".plan", ".project", ".pgpkey", and ".foward" in their home directory:
```
Comment (agnivade) [inline pages/common/finger.md]: This part can be removed `as well as contents of files ".plan", ".project", ".pgpkey", and ".foward" in their home directory` and just put `etc.` in place of it.

File: pages/common/finger.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+- Display information about currently logged in users:
+
+`finger`
+
+- Display information about a specific user:
+
+`finger {{username}}`
+
+- Display the user's login name, real name, terminal name and write status (as a "\*" after the terminal name if write permission is denied), idle time, login time, office location and office phone number:
```
Comment (agnivade) [inline pages/common/finger.md]: Remove - `write status (as a "\*" after the terminal name if write permission is denied), idle time, login time, office location and office phone number` and just put `other information`.

File: pages/common/finger.md (hunk 4 above)
Comment (agnivade) [inline pages/common/finger.md]: Produce _a_ multi-line _output_ format

Comment (MasterOdin) [reply pages/common/finger.md]: Done.

File: pages/common/finger.md (hunk 5 above)
Comment (MasterOdin) [reply pages/common/finger.md]: Done.

File: pages/common/finger.md (hunk 4 above)
Comment (MasterOdin) [reply pages/common/finger.md]: differs from the man page description, but I agree that it's clearer this way.

Comment (agnivade) [reply pages/common/finger.md]: Yes, the whole point is to differ from the man page. :smile:

**Pull request merged**